examples/README.md

```diff
@@ -20,11 +12,16 @@ Remember to replace the value of `ETHEREUM_PRIVATE_KEY` below with your own priv
 
 ### Running directly
 
+All commands below are to be run from `examples` directory.
+
 To run it directly, you will need [Python 3.9+](https://www.python.org/downloads/).
 
 ```bash
-pip install -r requirements.txt
-ETHEREUM_PRIVATE_KEY=private_key python example.py
+brew install python3.9 # i.e. ensure python3.9 is available
```

Comment:
this assumes user uses MacOS? May be leave generic "Make sure you have python V3.9 installed on your computer" ?